app/Domain/Integrations/IntegrationType.php

```diff
@@ -10,4 +10,15 @@ enum IntegrationType: string
     case SearchApi = 'search-api';
     case Widgets = 'widgets';
     case UiTPAS = 'uitpas';
+
+    public static function fromName(string $name): self
```

Comment:
The new `enum` class has a built-in method to do this, see https://www.php.net/manual/en/class.backedenum.php

Reply:
Are you talking about "from()"? That matches on value, not on name like I have.
And the reason I have to use this is because the value comes from another Enum, IntegrationExpirationTime, which has the name not the value as case name.
And the reason I do not use the value there is because these contain dashes (eg. entry-api) making them invalid case names.

Reply:
Now that I read the code again, this is maybe just the conversion from `IntegrationExpirationTime` to `IntegrationType`. That also made me wonder, as discussed through Slack, if the `IntegrationExpirationTime` doesn't miss entries for `UiTPAS` and `EntryAPI`

Reply:
Removed `IntegrationExpirationTime` and moved it into config.
Also makes it possible to drop the akward conversion of `IntegrationExpirationTime` to `IntegrationType`